internal/test/test_helpers/utilities.bash

```diff
@@ -42,7 +42,8 @@ redact_json() {
 		jq --sort-keys '
 			del(
 				.resourceSpans[].scopeSpans[].spans[].startTimeUnixNano,
-				.resourceSpans[].scopeSpans[].spans[].endTimeUnixNano
+				.resourceSpans[].scopeSpans[].spans[].endTimeUnixNano,
+				(.resourceSpans[].scopeSpans[].spans[].attributes[] | select(.key == "net.peer.port"))
```

Comment:
If the peer port is changed into something like the trace/span IDs (i.e. "******") we will still be able to evaluate they are included. By removing them, as is done here, it is removing the ability to test the attribute is added.